guides/common/modules/con_about-running-jobs-on-hosts.adoc

```diff
@@ -8,9 +8,12 @@ For custom Ansible roles that you create, or roles that you download, you must i
 Before you can use Ansible roles, you must import the roles into {Project} from the {SmartProxy} where they are installed.
 
 Communication occurs through {SmartProxyServer}, which means that {ProjectServer} does not require direct access to the target host, and can scale to manage many hosts.
-Remote execution uses the SSH service that must be enabled and running on the target host.
+
+On {SmartProxies} in `ssh` mode, remote execution uses the SSH service that must be enabled and running on the target host.
 Ensure that the remote execution {SmartProxy} has access to port 22 on the target hosts.
 
+On {SmartProxies} in `pull-mqtt` mode, remote execution uses MQTT to publish jobs it receives from {ProjectServer}, and the host subscribes to these jobs using the `yggdrasil` pull client.
+
```

Comment:
Nitpick inconsistency: for `ssh` mode, we describe the requirements here ("Ensure that the remote .." 3 lines above), for `pull-mqtt` we don't describe it here as it is longer. I think it is fine as is for the sake of clarity, maybe sentence like "Requirements for the `pull-mqtt` mode are described below / in Section [link]" can be added - leaving on @jeremylenz's decision.